### ISSUE TITLE
debugger works only after rails server restart

### ISSUE THREAD

**Issue by ashwintastic** [deivid-rodriguez/byebug#534]
## Problem description

  rails by-pass `debugger` I need to restart rails server so that `debugger` can come in action

## Expected behavior

Describe here how you expected byebug to behave in this particular situation.

## Actual behavior

Describe here what actually happened.

## Steps to reproduce the problem

This is extremely important! Providing us with a reliable way to reproduce
a problem will expedite its solution.

**Comment by shaicoleman** [comment]
Probably duplicate of #452
